badges/badges/settings.py

```diff
@@ -205,4 +205,12 @@ def skip_suspicious_operations(record):
 ##################################################################
 # Testing mode
 #################################################################
-TEST = 'test' in sys.argv
\ No newline at end of file
+TEST = 'test' in sys.argv
```

Comment:
Yes, let's chat about this. I have put this here, so I can see if system is in testing env for selenium use. Since I have run the tests and saw they apparently fail, I think it would be best to fix this properly.